.pylintdict

```diff
@@ -160,6 +161,7 @@ itertools
 iz
 jb
 jcf
+Jernigan
```

Comment:
I think we should use all lower-case chars in general because otherwise spelling checks become case-sensitive